ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java

```diff
@@ -21,20 +22,19 @@
     private HeaderPage headerPage;
     private ManageFormsPage manageForm;
     private ClinicianFacingPatientDashboardPage patientDashboardPage;
-
+    
     @Before
     public void setUp() throws Exception {
-
-        homePage = new HomePage(page);
-        assertPage(homePage);
```

Comment:
The function `assertPage()` is vital

Add `waitForPage()` function on `homePag`e object to ensure the page is loaded for assertion to be effective ie `assertPage(homePage.waitForPage()); `otherwise the function asserts an empty page resulting into `java.lang.AssertionError.`